test/unit/test_intrahost.py

```diff
@@ -224,8 +224,10 @@ def add_indel(self, sample, chrom, pos, acounts, libinfo=None):
 
     def dump_isnv_tmp_file(self, sample):
         fn = util.file.mkstempfname('.txt')
+        print("self.isnvs[sample]",sample,list(map(str,self.isnvs[sample])))
```

Comment:
are we sure we want to `print` all this or was this leftover from a debugging stage?

Reply:
leftover, whoops!